spec/helpers/pageflow/config_helper_spec.rb

```diff
@@ -38,6 +38,12 @@ module Pageflow
 
         expect(result['availablePublicLocales']).to eq(['fr'])
       end
+
+      it 'includes edit lock interval' do
+        result = JSON.parse(helper.editor_config_seeds)
+
+        expect(result['editorLockInterval']).to eq(Pageflow.config.editor_lock_interval)
```

Comment:
On second thought, I'm not sure how Rails' `15.seconds` actually transform into JavaScript numerals. So just to be sure everything works as expected, I'd suggest assigning the config option at the top of the test to, say, `5.seconds` and then assert that the rendered value actually is `5`. Since the default unit of interval duration in JS is milliseconds, I'd even consider calling the the JS seed property `editLockIntervalInSeconds`. That will also make the `* 1000` above much clearer.